.buildkite/pipeline_pr.py

```diff
@@ -68,6 +68,15 @@
     for step in kani_grp["steps"]:
         step["label"] = "🔍 Kani"
 
+if any(x.parent.name == "hiding_ci" for x in changed_files):
+    pipeline.build_group_per_arch(
+        "🕵️ Build Secret Hiding Kernel",
+        pipeline.devtool_test(
+            pytest_opts="-m nonci integration_tests/build/test_hiding_kernel.py",
```

Comment:
mh, I guess as long as we don't setup an nightly pipelines, reusing `nonci` is fine, but if we ever have a nightly pipeline on this branch that also passes `-m nonci`, then it might pick up this test by accident, which we don't want probably. Thought on using a dedicated pytest mark for this test? :o

Reply:
Added a new secret_hiding mark for the test